module-system/module-implementations/examples/sov-election/Cargo.toml

```diff
@@ -24,7 +24,7 @@ thiserror = { workspace = true }
 borsh = { workspace = true, features = ["rc"] }
 hex = { workspace = true }
 jsonrpsee = { workspace = true, features = ["macros", "client-core", "server"], optional = true }
-
+clap = {workspace = true, features = ["derive"]}
```

Comment:
Nit, formatting
```suggestion
clap = { workspace = true, features = ["derive"] }
```

Reply:
Can we make this optional and gate it behind the native flag?